src/frontend/context/Vilkårsvurdering/test/validering.test.ts

```diff
@@ -0,0 +1,59 @@
+import { Valideringsstatus } from '@navikt/familie-skjema';
+
+import { kjørValidering } from '../validering';
+import { mapFraRestPersonResultatTilPersonResultat } from '../vilkårsvurdering';
+import { genererPerson, genererPersonresultat } from './TestData';
+
+describe('vilkårsvurdering/validering', () => {
```

Comment:
Hva er målet med denne testen? Hvis det er å integrasjonsteste skjemaet tenker jeg det ser bra ut, men kunne det gitt like stor verdi å teste direkte på valideringsfunksjonene så slapp man å sette opp så mye domene-spesifikt for testene?

Reply:
Helt enig at det hadde vært enklere; årsaken til at vi tester gjennom kjørvalidering er at den er eksponeret og vi ønsker ikke å anpasse synlighet etter tests behov.